_projects/10-scala-native.md

```diff
@@ -8,7 +8,7 @@ contributors: [martinduhem, guillaume, ergys]
 status: Active
 type: project
 active: true
-priority: 1
+priority: 2
```

Comment:
What does this affect, actually?

Reply:
should just make it higher in the list than SIP